### PR TITLE
Noah/feedback system

### PR DESCRIPTION
**Pull request by NoahNok**
Can someone figure out why the tests are broken, I don't really understand?

### CODE REVIEW

File: main/urls.py [hunk 1]
```diff
@@ -1,13 +1,329 @@
...
+        if isinstance(o, Model):
+            return model_to_dict(o)
+        return super().default(o)
+
+
+class FormValidator(forms.Form):
+    jsonData = forms.JSONField()
+
+
+class FeedbackFormReturn(TemplateView):
```
Comment (joshdavies14) [inline main/urls.py]: Is there a specific reason these views are here and not in `views.py`? Just asking OOI :P

**Review by joshdavies14 — APPROVED** [review]
All LGTM, besides my one question OOI :)

File: main/urls.py (hunk 1 above)
Comment (NoahNok) [reply main/urls.py]: I'm moving them into there own area now, forgot to do this

Comment (joshdavies14) [reply main/urls.py]: No problemo! :)

Comment (NoahNok) [reply main/urls.py]: Have resolved these issues

**Pull request merged**